src/DynamoUtilities/PathHelper.cs

```diff
@@ -88,5 +90,64 @@ public static bool HasWritePermissionOnDir(string folderPath)
 
             return writeAllow && !writeDeny;
         }
+
+        /// <summary>
+        /// This is a utility method for checking if given path contains valid XML document.
+        /// </summary>
+        /// <param name="path">path to the target xml file</param>
+        /// <param name="xmlDoc">System.Xml.XmlDocument repensentation of target xml file</param>
+        /// <returns></returns>
+        public static bool isValidXML(string path, out XmlDocument xmlDoc)
+        {
+            // Based on https://msdn.microsoft.com/en-us/library/875kz807(v=vs.110).aspx
+            // Exception thrown indicate invalid XML document path 
+            try
+            {
+                xmlDoc = new XmlDocument();
+                xmlDoc.Load(path);
+                return true;
+            }
+            catch
+            {
+                xmlDoc = null;
+                return false;
+            }
+        }
+
+        /// <summary>
+        /// This is a utility method for checking if given path contains valid Json document.
+        /// </summary>
+        /// <param name="path">path to the target json file</param>
+        /// <param name="fileContents"> string contents of target json file</param>
+        /// <returns></returns>
+        public static bool isValidJson(string path, out string fileContents)
+        {
+            fileContents = File.ReadAllText(path);
+            fileContents = fileContents.Trim();
+            if ((fileContents.StartsWith("{") && fileContents.EndsWith("}")) || //For object
+                (fileContents.StartsWith("[") && fileContents.EndsWith("]"))) //For array
+            {
+                try
+                {
+                    var obj = Newtonsoft.Json.Linq.JToken.Parse(fileContents);
+                    return true;
+                }
+                catch (JsonReaderException jex)
+                {
+                    //Exception in parsing Json
+                    Console.WriteLine(jex.Message);
```

Comment:
these console writeLines will never make it to the dynamo console where we need them.

Reply:
hmm.. That was part of my purpose that I do not want to re-throw it and make client consume it everytime calls it, since these ex are easily reproducable once user gives us file, so I left these ex print here instead of DynamoModel..